grammars/coffeescript.cson

```diff
@@ -102,11 +102,10 @@
     'endCaptures':
       '0':
         'name': 'punctuation.definition.string.end.coffee'
```

Comment:
This should be changed to `punctuation.definition.embedded-javascript.end.coffee`, and similar for the beginCapture.